### PR TITLE
Rename ltp package to follow upstream changes

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Review by jlausuch — APPROVED** [review]

**Review by pdostal — APPROVED** [review]

**Review by grisu48 — APPROVED** [review]

File: tests/publiccloud/run_ltp.pm [hunk 1]
```diff
@@ -86,6 +86,7 @@ sub run {
     my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/stable/' . generate_version("_") . '/');
     my $provider;
     my $instance;
+    my $ltp_package = 'ltp-stable';
```
Comment (czerw) [inline tests/publiccloud/run_ltp.pm:89]: We use `LTP_PKG` variable  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/1a35043d7dac67f2f376d78ac4da1c78173196f3/tests/kernel/install_ltp.pm#L276  or default.

Comment (b10n1k) [inline tests/publiccloud/run_ltp.pm:89]: just a small thing. I dont see any good reason to define the package here. Would it make sense to have it public for this module? in such case the `gen_ltp_env` wouldnt need to change. wdyt?

Comment (asmorodskyi) [reply tests/publiccloud/run_ltp.pm:89]: this tests are running for 2-3 years and during this period there was never the case that I need to use different package here. On other side we have sooo many variables so each another variable especially when there is 0 chances  to actually use this variable

**Review by b10n1k — APPROVED** [review]
thanks @asmorodskyi.

**Pull request closed without merging**